Source: https://github.com/ungoogled-software/ungoogled-chromium-debian/issues/7
Upstream: No
Reason: Fixes build issue after updrade to libstdc++:9

In file included from gen/chrome/browser/ui/ui_jumbo_6.cc:14:                                                                                                                                                      
./../../chrome/browser/ui/tabs/tab_strip_model_observer.cc:51:22: error: defaulting this default constructor would delete it after its first declaration                                                           
TabStripModelChange::TabStripModelChange() = default;
../../chrome/browser/ui/tabs/tab_strip_model_observer.h:106:28: note: default constructor of 'TabStripModelChange' is implicitly deleted because field 'deltas_' of const-qualified type 'const std::vector<Delta>'
 would not be initialized                                                                                                                                                                                          
  const std::vector<Delta> deltas_;


diff --git a/chrome/browser/ui/tabs/tab_strip_model_observer.h.old b/chrome/browser/ui/tabs/tab_strip_model_observer.h
index 6c8bce5..7f67b7d 100644
--- a/chrome/browser/ui/tabs/tab_strip_model_observer.h.old
+++ b/chrome/browser/ui/tabs/tab_strip_model_observer.h
@@ -135,7 +135,7 @@ class TabStripModelChange {
 
  private:
   const Type type_ = kSelectionOnly;
-  const std::vector<Delta> deltas_;
+  std::vector<Delta> deltas_;
 
   DISALLOW_COPY_AND_ASSIGN(TabStripModelChange);
 };
